draft-ietf-rats-corim.md

```diff
@@ -255,8 +256,20 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 ~~~
 
 # CoRIM
+Concise Reference Integrity Manifests (CoRIM) contain tags that describe the
+composition and measurements of a platform, device, component, or software.
+CoRIM is an envelope to carry information typically exchanged between various supply chain entities and a Verifier. For example a Reference Value provider will use CoRIM to provision Reference Values for a specific Attester Class inside a Verifier. Typical Reference Values can be firmware revision or a measurement hash for the given firmare component. Similary an Endorsed Value Provider will use CoRIM to provision a specific Endorsed Value inside a Verifier. Endorsed Values may be any information such as Test Results or a certification data pertaining to an Attester.
+
+A CoRIM contains either CoMID and/or CoSWID tags. Please refer COMID and COSWID sections for further details.
```

Comment:
The first sentence restates what was said above and excludes CoBOM.
References to other sections should include a pointer.